### PR TITLE
Race and Ethnicity Fields added to Client Information 

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**

### CODE REVIEW

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+  componentWillReceiveProps(nextProps) {
+    let childClient = nextProps.childClient;
+    const hispanic = this.valueToString(childClient.hispanic_origin_code);
+    const selected = this.icwaToString(childClient.icwa_eligibility_code);
+    this.setState({ childClient, hispanic, selected });
+  }
+
+  handlePrimaryRace = name => ({ value }) => {
+    const { childClient } = this.state;
+    const a = RACE[25].value;
```
Comment (kelfink) [inline app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: what's this magic number?

Comment (RamuKammagani-osi) [reply app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: That's the Value from Constant RACE imported into the dropdown options of Primary Race

Comment (kelfink) [reply app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: You are grabbing the 25th race from this array?  I'm not even sure where the array is defined, but item 25 seems arbitrary.  
If 25 means "unable to determine". then add a RACE_UNABLE_TO_DETERMINE = 25 constant in the constants file, and refer to that index here, for the sake of clarity.

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        hispanic = [];
+    }
+    return hispanic;
+  };
+
+  icwaToString = codeToString => {
+    let selected;
+    switch (codeToString) {
+      case 'Y':
+        selected = ICWA_ELIGIBLE[0];
```
Comment (kelfink) [inline app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: If ICWA_ELIGIBLE were a has table of 'Y' => 'Yes', 'N','No', 'U' => 'Unknown', 'X' => (did not ask I think). this code would be much more readable and in fact would just be a lookup.  selected = ICWA[codeToString].

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    let childClient = nextProps.childClient;
+    const hispanic = this.valueToString(childClient.hispanic_origin_code);
+    const selected = this.icwaToString(childClient.icwa_eligibility_code);
+    this.setState({ childClient, hispanic, selected });
+  }
+
+  handlePrimaryRace = name => ({ value }) => {
+    const { childClient } = this.state;
+    const a = 'Unable to determine*';
+    let unableToDetermine;
```
Comment (edgibbs) [inline app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: can remove a and just check the string

**Pull request merged**